### DISCUSSION TITLE
Ughhhhhhhh dev stuff dev stuff dev stuff

### DISCUSSION THREAD

**Discussion by PatheticMustan** [Prodigy-Hacking/ProdigyMathGameHacking#462] [(D) Developer]
@ArcerionDev if you could learn more JS stuff that'd be great

having left WCM alone for a long time, many cheats have been broken

**Comment by ArcerionDev** [comment]
alright 
<strike>I can't get webpack to work</strike>

**Reply by ghost** [reply]
neither can i
